packages/ui/src/machines/authenticator/actors/signIn.ts

```diff
@@ -549,6 +554,19 @@ export function signInActor({ services }: SignInMachineOptions) {
         shouldSetupTOTP: (_, event): boolean => {
           return isExpectedChallengeName(getChallengeName(event), 'MFA_SETUP');
         },
+        shouldSelectMfaType: (_, event): boolean => {
```

Comment:
`isSelectMFATypeChallenge`, or `isSelectMFAChallenge` would make more sense. 

We would use `shouldSelectMFAType` if we had a mfa selector screen to redirect users to, but we do not yet.

Reply:
Done!

Reply:
I think `cond` name still needs to be updated 👍

Reply:
Ok. Done!